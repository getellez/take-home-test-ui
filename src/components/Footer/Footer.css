.Footer {
  font-weight: bold;
  text-align: center;
  padding: 50px;
  border-top: 1px solid rgba(0,0,0,0.1)
}