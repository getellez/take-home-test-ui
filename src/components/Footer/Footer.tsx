import './Footer.css'

export const Footer = () => {
  return (
    <footer className="Footer">
      <p>By GermÃ¡n </p>
    </footer>
  )
}
