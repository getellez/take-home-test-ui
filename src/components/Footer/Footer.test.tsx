import React from 'react'
import '@testing-library/jest-dom/extend-expect'
import { render, screen } from '@testing-library/react'
import { Footer } from './Footer'

describe('<Footer />', () => {
  
  const text = 'By Germán'

  test('Should match the snapshot', () => {
    const { container } = render(<Footer />)
    expect(container).toMatchSnapshot()
  })
  
  test('Should have the author name', () => {
    const { container } = render(<Footer />)
    expect(screen.getByText('By Germán')).toBeTruthy()
  })
  
})