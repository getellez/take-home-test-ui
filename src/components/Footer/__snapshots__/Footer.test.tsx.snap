// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> Should match the snapshot 1`] = `
<div>
  <footer
    class="Footer"
  >
    <p>
      By Germ√°n 
    </p>
  </footer>
</div>
`;
